tests/queries/0_stateless/03601_temporary_views.sql

```diff
@@ -0,0 +1,26 @@
+DROP TEMPORARY TABLE IF EXISTS tview_basic;
+DROP TEMPORARY TABLE IF EXISTS t_src;
+
+CREATE TEMPORARY TABLE t_src (id UInt32, val String) ENGINE = Memory;
+INSERT INTO t_src VALUES (1,'a'), (2,'b'), (3,'c');
+
+CREATE TEMPORARY VIEW tview_basic AS
+SELECT id, upper(val) AS u
+FROM t_src
+WHERE id <= 2;
+
+SELECT * FROM tview_basic ORDER BY id;
+
+EXISTS TEMPORARY VIEW tview_basic;
+SHOW TEMPORARY VIEW tview_basic;
+
+CREATE TEMPORARY VIEW IF NOT EXISTS tview_basic AS SELECT 0;
+
+-- CREATE OR REPLACE TEMPORARY VIEW tview_basic AS SELECT 0; -- { serverError 62 }
+
+CREATE TEMPORARY VIEW default.tview_db AS SELECT 1; -- { serverError 442 }
```

Comment:
Consider using error code names everywhere, so not `442` but `BAD_DATABASE_FOR_TEMPORARY_TABLE`